src/lib.rs

```diff
@@ -548,48 +553,28 @@ pub fn continue_run(
 
     let ratio = (new_num_rows as f64) / (original_num_rows as f64);
 
-    println!(
-        "Number of rows after compression: {} ({:.2}%)",
-        new_num_rows,
-        ratio * 100.
-    );
-
-    println!("Compression Statistics:");
-    println!(
-        "  Number of forced resets due to lacking prev: {}",
-        compressor.stats.resets_no_suitable_prev
-    );
-    println!(
-        "  Number of compressed rows caused by the above: {}",
-        compressor.stats.resets_no_suitable_prev_size
-    );
-    println!(
-        "  Number of state groups changed: {}",
-        compressor.stats.state_groups_changed
-    );
```

Comment:
Why are we removing these?

Reply:
I didn't actually mean to add them earlier! The idea is that all this already being given in the `ChunkStats` struct (what `continue_run` returns) and so the caller can do the output they want.

Reply:
Aha! Didn't realise these had been added

Reply:
They're still in `run()` (what the manual command line tool triggers) but I remove them here from `continue_run()`